Multisig/Logic/Models/Safe.swift

```diff
@@ -38,6 +38,10 @@ extension Safe: Identifiable {
         URL(string: "https://etherscan.io/address/\(address)")!
     }
 
+    func isDefaultFallbackHandler() -> Bool {
+        return fallbackHandler == App.shared.defaultFallbackHandler?.hex(eip55: true)
```

Comment:
ok, I see this is done here, good, but now we are again dependent on the server-returned formatting. It would be better to compare addresses together, not strings:

```
Address(fallbackHandler) == App.shared.defaultFallbackHandler
```